sysutils/virtualbox/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		virtualbox
+PLUGIN_VERSION=		1.0
+PLUGIN_COMMENT=		VirtualBox additions
+PLUGIN_DEPENDS=		virtualbox-ose-additions-nox11
+PLUGIN_MAINTAINER=	project@opnsense.org
```

Comment:
you should add your mail here or yield to franco@opnsense.org -- "shared" mail accounts are never a good idea for reaching a maintainer

Reply:
Ok, I'd prefer to insert your contact here if it's ok for you.

Reply:
Sure, no problem

Reply:
Just did it with andrewhotlab/opnsense-plugins@5311b9f